ban/http/api.py

```diff
@@ -173,11 +173,20 @@ def get_resource(self, identifier):
 
         parameters:
             - $ref: '#/parameters/identifier'
+              name: identifier
```

Comment:
Flex needs that, in this case, a name must be defined.
If not, this is what error message shows :
```self = defaultdict(<class 'flex.exceptions.ErrorList'>, {'paths': [defaultdict(<class 'flex.exceptions.ErrorList'>, {'/group/..., {'required': [defaultdict(<class 'flex.exceptions.ErrorList'>, {'name': ['This value is required']})]})})]})]})]})]})```